src/Promitor.Integrations.Sinks.Statsd/StatsdMetricSink.cs

```diff
@@ -31,16 +39,24 @@ public async Task ReportMetricAsync(string metricName, string metricDescription,
             Guard.NotNull(scrapeResult.MetricValues, nameof(scrapeResult.MetricValues));
 
             var reportMetricTasks = new List<Task>();
+            var formatterType = _statsDConfiguration.CurrentValue?.MetricFormat ?? StatsdFormatterTypesEnum.Default;
 
             foreach (var measuredMetric in scrapeResult.MetricValues)
             {
                 var metricValue = measuredMetric.Value ?? 0;
 
-                var reportMetricTask = ReportMetricAsync(metricName, metricDescription, metricValue, new Dictionary<string, string>());
-                reportMetricTasks.Add(reportMetricTask);
-            }
+                switch (formatterType)
```

Comment:
We should introduce startup validation to block Geneva format without configuring an account or namespace